.gitignore

```diff
@@ -1,58 +1,11 @@
-
-# Created by https://www.gitignore.io/api/java,gradle,intellij,eclipse
-
-### Intellij ###
-# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
-# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
-
-# User-specific stuff:
-.idea/workspace.xml
-.idea/tasks.xml
-
-# Sensitive or high-churn files:
-.idea/dataSources/
-.idea/dataSources.ids
-.idea/dataSources.xml
-.idea/dataSources.local.xml
-.idea/sqlDataSources.xml
-.idea/dynamic.xml
-.idea/uiDesigner.xml
-
-# Gradle:
-.idea/gradle.xml
-.idea/libraries
-
-# Mongo Explorer plugin:
-.idea/mongoSettings.xml
-
-## File-based project format:
+# Some contributors are using IntelliJ, but decided to configure
+# the project individually, since checking in IntelliJ related
+# files leads to annoying recurring conflicts/changes.
+.idea
 *.iws
+*.iml
 
-## Plugin-specific files:
-
-# IntelliJ
-/out/
```

Comment:
Why is the out directory no longer ignored? During my tests, I think I saw it once created, but I am not sure what caused it.

Reply:
Because I decided to remove the default set of ignores from gitignore.io. But sure, we can include `/out/` if you like ...